### ISSUE TITLE
RD AD review: unclear about which interactions need what Security Considerations

### ISSUE THREAD

**Issue by mcr** [ietf-rats-wg/architecture#367]
````

** Section 12.1.1.  Typo. s/Sometimes this is/Sometimes this/


````

**Comment by mcr** [comment]
````
** Section 12.2

Any solution that conveys information used for security purposes,
   whether such information is in the form of Evidence, Attestation
   Results, Endorsements, or appraisal policy ...

Should reference values be included?

** Section 12.2

   It is also important that the appraisal policy was itself obtained
   securely.  If an attacker can configure appraisal policies for a
   Relying Party or for a Verifier, then integrity of the process is
   compromised.

In addition to reconfiguration, wouldn't the ability to intercept either the appraisal policy for evidence or attestation results potentially provide the attacker insight into defensive mitigations?  This suggests that this policy might need confidentiality protection.
````

**Comment by mcr** [comment]
````
** Section 12.4.  Is there any read sensitivity for the trust anchor store?

** Section 12.4.  Since RFC5280 is being invoked here, is there an expectation that certificates in RATS would confirm to this profile?
````

**Comment by nedmsmith** [comment]
Section 12.1.1
Is it more accurate to say that an "A foundational security assumption of a RATS architecture is that a Relying Party is able to trust the Attester even if the user or owner is not trusted"

I think the statement is closer to "A foundational security assumption of a RATS architecture it that a Relying Party is able to establish the basis for trusting the Attester functions, or Attester users.

**Comment by mcr** [comment]
fixed in cae9604
